cli/common/utils.go

```diff
@@ -564,6 +564,36 @@ func ParseDependencies(rawDeps []string) (PackDependencies, error) {
 	return deps, nil
 }
 
+func isCommandAvailable(command string) bool {
+	commandName := strings.Split(command, " ")[0]
+
+	_, err := exec.LookPath(commandName)
+
+	if err != nil {
+		return false
+	}
+	return true
+}
+
+func ParseUserInstallScript(instScript string) (string, error) {
```

Comment:
I understood your idea, but I also think that if a user wants to specify his own pre- or post-install script, he is clever enough to write it appropriately. I think we should simply add a note in the doc, that all executable's paths should be absolute, or `/bin/sh -c ""` should be used.

I mean, he should use one of the following

```
/bin/sh -c 'mkdir dir-path'
# or
/bin/mkdir dir-path
```

The idea of using `/bin/sh -c` is to delegate discovering executables to `sh`. It's used since on different systems it can be placed in different paths.